src/base/static/js/views/survey-view.js

```diff
@@ -82,6 +87,22 @@
         }, 700);
       }
 
+      if (this.options.placeDetailView.isEditingToggled) {
+        var editEvents = "keyup";
+        $.each(this.$el.find(".responses form"), function() {
+          $(this).on(editEvents, function(e) {
```

Comment:
Ditto about the specific list. And if this is in more than one place, we should certainly factor it out.